src/main/java/com/epam/ta/reportportal/dao/LaunchRepositoryCustomImpl.java

```diff
@@ -151,4 +157,24 @@ public Boolean identifyStatus(Long launchId) {
 				.orderBy(LAUNCH.NAME, LAUNCH.NUMBER.desc())
 				.fetchOneInto(Launch.class));
 	}
+
+	@Override
+	public Page<Launch> findAllLatestByFilter(Filter filter, Pageable pageable) {
+
+		return PageableExecutionUtils.getPage(
+				LAUNCH_FETCHER.apply(dsl.fetch(dsl.with(LAUNCHES)
+						.as(QueryBuilder.newBuilder(filter).with(pageable).build())
```

Comment:
withLatest? 
check that with latest works only for launches, but we can call it for any filter target and it wouldn't work
think about how to avoid it. for example add some checks